hawkular-alerts-rest-tests/pom.xml

```diff
@@ -147,6 +165,19 @@
               </execution>
             </executions>
           </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>wildfly</id>
```

Comment:
Travis by default uses -Prest profile, so if we use -Prest for local-up-and-running and -Pwildfly for the all-automated test, we need to update also the .travis.yml file to point to this profile.